### PR TITLE
Updated/merged team a course content bl

### PR DESCRIPTION
**Pull request by scottmcglynn**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,10 +25,15 @@ doc/api/
 
 .flutter-plugins
 .flutter-plugins-dependencies
+<<<<<<< HEAD
```
Comment (AlmostCarbon-14) [inline .gitignore]: Please remove these merge conflict vestiges

File: teamA/README.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
+<<<<<<< HEAD
```
Comment (AlmostCarbon-14) [inline teamA/README.md]: Please remove these merge conflict vestiges

File: .gitignore (hunk 1 above)
Comment (scottmcglynn) [reply .gitignore]: Fixed.

File: teamA/README.md (hunk 2 above)
Comment (scottmcglynn) [reply teamA/README.md]: Fixed.

**Review by AlmostCarbon-14 — APPROVED** [review]

**Pull request merged**